### PR TITLE
[Rompetrol] Namespace ref

### LINKED ISSUE #5932: Rompetrol - missing stations
There are about 80 Romanian stations and about 30 Bulgarian stations which are missing. It seems that the problem is caused by duplicated `ref`s on their end (the databases seem to be independent so each site has station with `ref=1`, `ref=2` and so on). Is it possible to resolve that issue? Possibly without having 4 almost identical spiders?

### PR DESCRIPTION
**Pull request by Cj-Malone**
Fixes #5932

### CODE REVIEW

**Comment by Dimitar5555** [comment]
Thank you @Cj-Malone. Unfortunately it looks like it's a bigger mess than I first thought. The Romanian site is returning stations in Bulgaria as well which means that that they are on the right coordinates but with the wrong `addr:country` tag. I haven't checked if the data is duplicated or not. Is it possible to fix it somehow?

**Comment by Cj-Malone** [comment]
It does seem like every BG location is in the RO data. Hows this look?

**Comment by Dimitar5555** [comment]
I don't see any missing pins when comparing the map here and the map on the Bulgarian website.

File: locations/spiders/rompetrol.py [hunk 1]
```diff
@@ -8,50 +8,38 @@ class RompetrolSpider(Spider):
...
-            if "14" in location["services"]:
-                apply_category(Categories.CHARGING_STATION, item)
-            if "33" in location["services"]:
-                apply_category(Categories.SHOP_LAUNDRY, item)
-            if "20" in location["services"]:
-                apply_category(Categories.COMPRESSED_AIR, item)
-            if "17" in location["services"]:
-                apply_category(Categories.RESTAURANT, item)
+
+            apply_yes_no("fuel:electric", item, "14" in location["services"])
```
Comment (Dimitar5555) [inline locations/spiders/rompetrol.py:40]: The number for EV chargers is 34. 14 is for AdBlue.

Comment (Cj-Malone) [reply locations/spiders/rompetrol.py]: Thanks.

**Pull request merged**